backends/tfhe-cuda-backend/src/cuda_bind.rs

```diff
@@ -786,4 +786,50 @@ extern "C" {
         mem_ptr: *mut *mut i8,
     );
 
+    pub fn scratch_cuda_integer_radix_sum_ciphertexts_vec_kb_64(
+        v_stream: *const c_void,
+        mem_ptr: *mut *mut i8,
+        glwe_dimension: u32,
+        polynomial_size: u32,
+        big_lwe_dimension: u32,
+        small_lwe_dimension: u32,
+        ks_level: u32,
+        ks_base_log: u32,
+        pbs_level: u32,
+        pbs_base_log: u32,
+        grouping_factor: u32,
+        num_blocks_in_radix: u32,
+        max_num_radix_in_vec: u32,
+        message_modulus: u32,
+        carry_modulus: u32,
+        pbs_type: u32,
+        allocate_gpu_memory: bool,
+    );
+
+    pub fn cuda_integer_radix_sum_ciphertexts_vec_kb_64(
+        v_stream: *const c_void,
+        result: *mut c_void,
+        radix_lwe_vec: *mut c_void,
+        num_radix_in_vec: u32,
+        mem_ptr: *mut i8,
+        bsk: *const c_void,
+        ksk: *const c_void,
+        num_blocks: u32,
+    );
+
+    pub fn cleanup_cuda_integer_radix_sum_ciphertexts_vec(
+        v_stream: *const c_void,
+        mem_ptr: *mut *mut i8,
+    );
+
+    /// Copy `size` memory asynchronously from `src` to `dest` on the same GPU `gpu_index` using
+    /// the Cuda stream `v_stream`.
+    // return next 1 byte pointer from dest
+    pub fn cuda_concatenate_device_memory_async(
```

Comment:
Again, not sure about this one, it seems to me it could be replaced by copy_gpu_to_gpu + ptr shift.